src/main/java/org/polyfill/services/JSONConfigLoaderService.java

```diff
@@ -0,0 +1,89 @@
+package org.polyfill.services;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.polyfill.interfaces.ConfigLoaderServiceInterface;
+import org.springframework.expression.ExpressionException;
+import org.springframework.stereotype.Service;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by bvenkataraman on 10/13/16.
+ */
+@Service
+public class JSONConfigLoaderService implements ConfigLoaderServiceInterface{
+
+
+    /**
+     * Converts the JSON file to a HashMap
+     *
+     * @param JsonConfigFile - config file in a JSON format
+     * @return map - HashMap which was converted from the JSON file
+     * @throws IOException
+     */
+    private HashMap<String, Object> convertJSONConfigToMap(File JsonConfigFile) throws IOException{
+        System.out.println(JsonConfigFile.isFile());
+        HashMap<String,Object> map = new ObjectMapper().readValue(JsonConfigFile, HashMap.class);
+        return map;
+    }
+
+    /**
+     * Retrieve JSON config file from specified path and convert it into a Map
+     *
+     * @param path - File path to retrieve the config file (ex. ./polyfills/Element/config.json)
+     * @return configMap - Returns the JSON config in a HashMap format
+     * @throws Exception
+     */
+    public HashMap<String, Object> getConfig(String path) throws Exception {
```

Comment:
Should we just return a ConfigObject here? This way, the ConfigObject doesn't need to have getConfig method, only getIn. This way we don't need to wire the ConfigObject to any config loader service and implement additional ConfigObject for future loader services.